site/mkdocs.yml

```diff
@@ -51,7 +51,7 @@ nav:
     - Flink: flink.md
     - Spark: spark.md
     - Spark Streaming: spark-structured-streaming.md
-    - Presto: presto.md
+    - Presto: https://prestosql.io/docs/current/connector/iceberg.html
```

Comment:
Now there are no links to "presto.md". We probably should remove it

Reply:
Oh I see we are leaving it in place for past links? Perhaps we should change it to a redirect